stratum/hal/lib/tdi/tdi_table_manager.cc

```diff
@@ -995,7 +995,11 @@ ::util::Status TdiTableManager::WriteRegisterEntry(
   absl::optional<uint32> register_index;
   if (register_entry.has_index()) {
     register_index = register_entry.index().index();
+  } else {
+    RETURN_ERROR(ERR_INVALID_PARAM) << "Register entry does not have "
```

Comment:
Would it be clearer to say "Invalid register entry index"?

"does not have a valid index" is a roundabout way of saying it